website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -313,6 +363,14 @@ deployment "static-client" successfully rolled out
 
 You can verify connectivity of the static-client and terminating gateway via a curl command:
 
+**For external services registered with the Consul catalog**
```

Comment:
```suggestion
**For external services registered with the Consul catalog**
```
I get that we're bolding this sentence fragment so that it reads like a label for the following command, but I wonder if it wouldn't be better to wrap a CodeBlockConfig element around it and use the heading=""" attribute to make it look nicer.

Reply:
OK, I'll try that out.

Reply:
I agree it looks nicer!